content/blog/2022/11/2022-11-14-hacktoberfest-recap.adoc

```diff
@@ -0,0 +1,176 @@
+This is his third Hacktoberfest, and first Hacktoberfest contributing to Jenkins.
+He has contributed to open source previously for astronomy, but shifted focus since graduating.
+His goals during Hacktoberfest were giving back to the Jenkins community and acquiring more software development skills along the way.
+
+image:/images/post-images/2022-11-14-hacktoberfest-recap/krisstern.png[Kris Stern, width=20%]
+
+== Returning Jenkins contributors
+
+link:https://github.com/jimklimov[Jim Klimov] is a regular Jenkins contributor.
+Jim is based in the Czech Republic, and has been working in open source for quite some time.
+He has worked on various enterprise document workflow and messaging systems, and is currently a DevOps architect.
+He is the creator and maintainer of several Jenkins plugins.
+Along with his own experience, he also learned a lot from previous contributors.
+Having this knowledge feels like standing on the shoulders of giants.
+This is his second Hacktoberfest in Jenkins, but Jim contributes all year round.
+Challenges that have come up for Jim include maintainability expectations to get pull requests merged, and architecture becoming more costly as time goes on.
+
+link:https://github.com/highflyer910[Teona Mushambadze] is another contributor who submitted work to Jenkins during Hacktoberfest.
+Teona created three new logos: link:/artwork/#turkey[Turkey], link:/artwork/#georgia[Georgia], and a link:/artwork/#nerd[Jenkins nerd] themed logo.
+These logos help connect Jenkins to users globally and extend the community even further.
+She is a freelance developer, and this is her first time contributing to Jenkins in Hacktoberfest.
+Her motivation when starting was curiosity and learning more about open source.
+Through her contributions, she has learned how to work with someone else's code.
+There have been challenges when new tools are installed, but don't work as expected.
+However, these are learning experiences, as each one is an opportunity to understand what is causing the behavior.
+
+image:/images/post-images/2022-11-14-hacktoberfest-recap/teona.png[Teona Mushambadze, width=20%]
+
+link:https://github.com/StefanSpieker[Stefan Spieker] is a solutions architecht and DevOps engineer from Aachen, Germany.
+He has been participating in Hacktoberfest since 2019, with a strong focus on Jenkins core and plugin development.
+Hacktoberfest was the trigger to start participating, and Stefan has continued to contribute regularly.
+Stefan likes to contribute to open source in his free time and has found that there are always ways to improve.
+For Hacktoberfest, this included updating projects that still have spotbugs disabled and adopting a plugin to become a maintainer.
+Since open-source software is used daily, especially in professional life, Stefan embraces this by giving back as an OSS consumer.
```

Comment:
```suggestion
Since he uses open-source software daily, especially in his professional life, Stefan embraces this by giving back as an OSS consumer.
```

Reply:
Thanks for catching that @kellie-freeman , I had been updating the pronoun usage to match the contributor, but must have missed that when changing things.